src/gwkokab/models/hybrids/_bp2pmultispinfull.py

```
@@ -0,0 +1,688 @@
+
+        self._support = all_constraint(
+            [
+                mass_sandwich(m2min, mmax),
+                constraints.unit_interval,
+                constraints.unit_interval,
+                constraints.interval(-1.0, 1.0),
+                constraints.interval(-1.0, 1.0),
+            ],
+            [(0, 2), 2, 3, 4, 5],
+        )
+        super(BrokenPowerlawTwoPeakMultiSpinMultiTilt, self).__init__(
```

Ruff (0.14.2)

Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)